packages/addons-v5/test/commands/addons/info.js

```diff
@@ -39,7 +39,7 @@ describe('addons:info', function () {
 Attachments:  acme-inc-www::DATABASE
 Installed at: Invalid Date
 Owning app:   acme-inc-www
-Plan:         heroku-postgresql:hobby-dev
+Plan:         heroku-postgresql:no-free
```

Comment:
This is minor, but should we use a real plan here (and all the other spots), like `mini`?

Reply:
+1